docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -303,13 +307,19 @@
         }
       }
 
-
-      &[aria-expanded="true"]::after {
-        @include hb-menu-minus-icon;
-
-        @include grid-media-min('lg') {
-          .js & {
-            @include hb-menu-up-arrow-icon;
+      // The minus and arrow up icons representing the expanded state of a nav item
+      // should only apply when item has the .hb-main-nav__item--parent class.
+      // This class is only added when there are children in the nested toggler.
+      .hb-main-nav__item--parent & {
```

Comment:
We should be able to combine lines 313 and 314 into:
```
.hb-main-nav__item--parent &[aria-expanded="true"]::after {
```

Then we can get rid of the `stylelint-disable` below.

Reply:
Oh Duh!!! 🤣  🤣  🤣  Good call!!